### DISCUSSION TITLE
How to change startup port?

### DISCUSSION THREAD

**Discussion by Cyberavater** [m0bilesecurity/RMS-Runtime-Mobile-Security#141] [Q&A]
5491 is already being used by MSI by default, and I want to start rms on a different port.

**Comment by m0bilesecurity** [comment]
Ciao @Cyberavater 

NOTE: Default RMS port has been changed from 5000 to 5491 because since MacOS Ventura, port 5000 is the Control Center's default port. If you wish to change the default port, you can also run RMS with the --port parameter followed by the desired port number e.g. rms --port 9000
